src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,6 +1,6 @@
 import './MovieCard.scss';
 
-export const MovieCard = ({ movie }) => (
+export const MovieCard = ({ key, movie }) => (
```

Comment:
No need to pass `key` prop)
```suggestion
export const MovieCard = ({ movie }) => (
```

Enough to set it here:
```
 {movies.map(movie => (
      <MovieCard key={movie.imdbId} movie={movie} />
 )}
 ```